src/doc/guide.md

```diff
@@ -414,23 +421,7 @@ We can add the type if we want to, though. Types come after a colon (`:`):
 let x: int = 5;
 ```
 
-If I asked you to read this out loud to the rest of the class, you'd say "`x`
-is a binding with the type `int` and the value `five`."
-
-In future examples, we may annotate the type in a comment. The examples will
-look like this:
-
-```{rust}
-fn main() {
-    let x = 5i; // x: int
-}
-```
-
-Note the similarities between this annotation and the syntax you use with `let`.
-Including these kinds of comments is not idiomatic Rust, but we'll occasionally
-include them to help you understand what the types that Rust infers are.
```

Comment:
I can surely mention that we may annotate types in comments, but I'd like to keep it short and sweet. The guide is for Rust newcomers, not programming newcomers. I'll clarify in comments below, if needed.